Lombiq.Hosting.Azure.ApplicationInsights/Services/TelemetryFilter.cs

```diff
@@ -99,4 +96,14 @@ private DependencyTelemetry GetDependencyTelemetryFailureToIgnore(ITelemetry ite
         // Name property value could be different depending on the environment, so using the Data property instead.
         return dependency.Data.Contains(mediaBlobStorageContainerName) ? dependency : null;
     }
+
+    [GeneratedRegex(@"Azure\.RequestFailedException: The specified container already exists\.",
+        RegexOptions.Compiled,
```

Comment:
`Compiled` is [not used in these cases](https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-source-generators#:~:text=The%20RegexOptions.Compiled%20flag%20is%20ignored%20by%20the%20source%20generator%2C%20thus%20making%20it%20no%20longer%20needed%20in%20the%20source%20generated%20version.).